Scrapping.py

```diff
@@ -0,0 +1,43 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+username=input('Enter Username')
+page=requests.get("https://www.codechef.com/users/"+username)
```

Comment:
Similar to temp.py. This is why I was getting IndexError due to CodeChef blocking this as a bot.
```suggestion
page=requests.get("https://www.codechef.com/users/"+username, headers={'User-Agent': 'Mozilla/5.0'})
```